flaml/automl.py

```diff
@@ -860,6 +862,9 @@ def cat_hp_cost(self) -> dict:
             for estimator in self.estimator_list:
                 c = self._search_states[estimator].cat_hp_cost
                 configs.append(c)
+            configs.append([

```

Comment:
add comments in docstr about this extra element appended?

Reply:
done.